test/functional/Java8andUp/playlist.xml

```diff
@@ -24,25 +24,6 @@
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<include>java8andUpSettings.mk</include>
-	<test>
-		<testCaseName>generalSanityTest_SE80</testCaseName>
-		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
-	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
-	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames generalTest \
-	-groups $(TEST_GROUP) \
-	-excludegroups $(DEFAULT_EXCLUDE); \
-	$(TEST_STATUS)</command>
-		<subsets>
-			<subset>SE80</subset>
-		</subsets>
-		<levels>
-			<level>sanity</level>
-		</levels>
-		<groups>
-			<group>functional</group>
-		</groups>
-	</test>
```

Comment:
How does removing this relate to the subject of this PR?

Reply:
This is to reduce duplication by combining generalSanityTest_SE80 with generalSanityTest. Noticed this as GPU sortTest  is defined by generalSanityTest.  I could create a separate PR if preferred.

Reply:
A separate PR would be fine, but if you update the title of this PR to reflect the full effect of the proposed changes, I'm fine with just this one.

Reply:
Thanks, I've updated the title.